client/src/leap/soledad/client/crypto.py

```diff
@@ -164,24 +161,16 @@ class SoledadCrypto(object):
     General cryptographic functionality encapsulated in a
     object that can be passed along.
     """
-    def __init__(self, soledad):
+    def __init__(self, secret):
         """
         Initialize the crypto object.
 
-        :param soledad: A Soledad instance for key lookup.
-        :type soledad: leap.soledad.Soledad
+        :param secret: The Soledad storage secret.
+        :type secret: str
         """
-        self._soledad = soledad
-
-    def encrypt_sym(self, data, key,
-                    method=crypto.EncryptionMethods.AES_256_CTR):
-        return encrypt_sym(data, key, method)
+        self.secret = secret
```

Comment:
It is actually used outside the class, the previous code had a `@property` getter. I could do the same, it might be a bit more pythonic.